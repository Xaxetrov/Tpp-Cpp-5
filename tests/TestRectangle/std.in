PC Poly1 10 10 10 20 15 25 30 20 30 10
R Poly2 -10 -10 0 0
R Poly1 -1 -1 20 10
R Poly3 -1 -1 20 10 25 68
R Poly4 20 20 11 11
EXIT
