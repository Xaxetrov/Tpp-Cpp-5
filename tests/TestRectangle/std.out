OK
OK
ERR
#Incorrect name
ERR
#Incorrect number of coordinates
OK

